hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AInstrumentation.java

```diff
@@ -1470,7 +1456,7 @@ public void blockUploadFailed(
     @Override
     public void bytesTransferred(long byteCount) {
       bytesUploaded.addAndGet(byteCount);
-      incrementGauge(STREAM_WRITE_BLOCK_UPLOADS_BYTES_PENDING, -byteCount);
+      incAllGauges(STREAM_WRITE_BLOCK_UPLOADS_BYTES_PENDING, -byteCount);
```

Comment:
nit: why change in name as still only one guage is being incremented in the method underneath.

Reply:
incAllGauges does both the iostats gauge and the Hadoop metrics gauges; we need that